### ISSUE TITLE
Receiving Swift Compile Error: 'MapboxAccounts/MapboxAccounts.h' file not found

### ISSUE THREAD

**Issue by roncodes** [homeeondemand/react-native-mapbox-navigation#12]
When I go to build the project after following the installation steps I receive a Swift Compile Error in xcode.

Xcode 12.3
RN 0.63.4

```
<module-includes>:1:9: note: in file included from <module-includes>:1:
#import "MapboxCoreNavigation-umbrella.h"
        ^
/../ios/Pods/Headers/Public/MapboxCoreNavigation/MapboxCoreNavigation-umbrella.h:13:9: note: in file included from /../ios/Pods/Headers/Public/MapboxCoreNavigation/MapboxCoreNavigation-umbrella.h:13:
#import "MapboxCoreNavigation.h"
        ^
/../ios/Pods/Headers/Public/MapboxCoreNavigation/MapboxCoreNavigation.h:9:9: note: in file included from /../ios/Pods/Headers/Public/MapboxCoreNavigation/MapboxCoreNavigation.h:9:
#import "MBXAccounts+CoreNavigationAdditions.h"
        ^
/../ios/Pods/Headers/Public/MapboxCoreNavigation/MBXAccounts+CoreNavigationAdditions.h:1:9: error: 'MapboxAccounts/MapboxAccounts.h' file not found
#import <MapboxAccounts/MapboxAccounts.h>
        ^
<unknown>:0: error: could not build Objective-C module 'MapboxCoreNavigation'
```

Any idea the cause of issue or if I'm missing a step?

**Comment by rossmartin** [comment]
Did you create an Objective-C bridging header? Here is the step - https://github.com/homeeondemand/react-native-mapbox-navigation#ios-specific-instructions

**Comment by rossmartin** [comment]
This issue has been resolved in the latest version (1.0.2) and `use_frameworks!` is no longer needed.

**Comment by rossmartin** [comment]
I think this issue might have been caused by not creating the bridging header. Can you please try the latest version (1.0.2) and review the updated ios specific instructions when installing?
